lib/capistrano/tasks/slackify.cap

```diff
@@ -7,10 +7,13 @@ namespace :slack do
 
       if fetch(:slack_notify_events).include? :started
         info 'Notifying Slack of deploy starting'
-
-        execute :curl, '-X POST', '-s', '--data-urlencode',
-          Slackify::Payload.build(self, :starting),
-          fetch(:slack_url)
+        fetch(:slack_channel).split(',').each {|channel|
+          pay = Slackify::Payload.new(self, :starting)
+          pay.channel = (channel)
```

Comment:
If we're going to pass the `channel` value into the `Payload` instance, would it be preferable to pass it as another argument on the initializer? Or alternatively as an argument to `build`?